.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
why do we need this file?

Reply:
we don't need this file. I have deleted this file